data/Republic Navy Intro.txt

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) <whenever> by Michael Zahniser
```

Comment:
Actually, since you wrote this whole thing, wouldn't the copyright belong to you?

Reply:
an example is the base game marauders.txt where instead of MZ it says "Copyright (c) 2016 by Iaz Poolar"
so yes, whoever actually wrote this should have their name (or username if unwanted) in the copyright file instead of MZ